app-antivirus/clamav/patches/haiku-1.2.0.patch

```diff
@@ -0,0 +1,43 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Try unpacking and patching with this one (not building) eg: `haikuporter clamav -b`
Then run `haikuporter clamav -e` This will extract the `patchset` for you with your credits added to it, then use `clamav-$portVersion.patchset` in the recipe :)